### ISSUE TITLE
[Feature Add] Add a new attribute to allow the plugin "html-button-response" to delay the display of the button

### ISSUE THREAD

**Issue by thtTNT** [jspsych/jsPsych#3289]
Hi JsPsych Amazing Team,

Recently, our project team asked us to delay the display of a button, 
For example, participants were asked to recall something and were only allowed to click the button after 30 seconds.

I think this is a pretty common request in psychology experiments, so I'd like to add an attribute to do this, what do you think?

I can do that, and if you allow me to do that, what would you like the attribute to be called?

**Comment by jodeleeuw** [comment]
Hey @thtTNT,

The way that I would do this currently is to stack two trials sequentially (one with a fixed `trial_duration` and no response option, the second with a button). But I can see the value in making this easier to do. 

Do you imagine having the buttons disabled or the buttons hidden? My instinct is that disabled is a better default, but if there's a good reason for them to be hidden I'm open to considering.

`enable_buttons_after` or `show_buttons_after` could be clear parameter names.

If you add this to `html-button-response` would you be willing to add it to all the other -button-response plugins for consistency?

**Comment by thtTNT** [comment]
Hi @jodeleeuw ,

Thank you for your prompt reply.

I'm working on "audio-button-response" and "video-button-response" but the problem is that they already have a attribute called "response_allowed_while_playing". So What I plan to do is that start the timer of delay only after the playback is over.

Please allow me to confirm this to you. If you don't think this is a good idea, please let me know.

**Comment by jodeleeuw** [comment]
That's tricky! I think if response_allowed_while_playing is true then the timer should start right away. Does that match your intuition?

**Comment by anasophiarc** [comment]
Hi @thtTNT I am a student who is new to jsPsych and I really want to use your attribute for a video button response trial (it would make it difficult to stack 2 trials seamlessly). Would u mind briefly explaining how I could  include your attribute into my work?

**Comment by thtTNT** [comment]
Hi @anasophiarc, so happy to see my work help you. It's a good start that checking the document of each plugin (https://www.jspsych.org/7.3/plugins/html-button-response/)
I don't think here is a good place to talk about this. Maybe you can create a new issue with more detail? That's really helpful for me.
